x-pack/plugins/observability_solution/synthetics/server/saved_objects/private_locations.ts

```diff
@@ -7,12 +7,34 @@
 
 import { SavedObjectsType } from '@kbn/core/server';
 import { modelVersion1 } from './migrations/private_locations/model_version_1';
-import { privateLocationsSavedObjectName } from '../../common/saved_objects/private_locations';
-export const privateLocationsSavedObjectId = 'synthetics-privates-locations-singleton';
+import {
+  legacyPrivateLocationsSavedObjectName,
+  privateLocationsSavedObjectName,
+} from '../../common/saved_objects/private_locations';
 
 export const PRIVATE_LOCATIONS_SAVED_OBJECT_TYPE: SavedObjectsType = {
   name: privateLocationsSavedObjectName,
   hidden: false,
+  namespaceType: 'multiple',
```

Comment:
Our private locations used to be `agnostic` and exist in all spaces. Why no are we changing to multiple? Is there a requirement for this to be excluded from specific namespaces?

Reply:
yes barclays and few other customers have asked that, so it's a good opportunity.